packages/nimble-components/src/toggle-button/template.ts

```diff
@@ -7,30 +8,35 @@ import {
 } from '@microsoft/fast-foundation';
 import type { ToggleButton } from '.';
 
+/* eslint-disable @typescript-eslint/indent */
```

Comment:
each eslint-disable should have a corresponding eslint-enable so future additions after the section don't accidentally have the rule disabled. Can you add an eslint-enable for the rule after the template definition?